tests/test_hmc.py

```diff
@@ -100,17 +96,27 @@ def logprob_fn(y):
     assert np.var(samples[1000:]) == pytest.approx(4.0, rel=1e-1)
 
 
-def assert_mcse(samples, true_param, p_val=0.01):
+def compute_ess(samples):
     d = arviz.convert_to_dataset(np.expand_dims(samples, axis=0))
-    ess = np.array(arviz.ess(d).to_array())
+    ess = arviz.ess(d).to_array()
+    return ess
+
+
+def compute_mcse(samples, true_param):
+    """Compute the probability of getting an error more extreme
+    than the one we observe assuming the samples are really drawn
+    from the target distirbution.
```

Comment:
Is this assuming a normal target distribution?

Reply:
Yes